docs/framework/api-architecture.md

```diff
@@ -0,0 +1,175 @@
+# API architecture
+
+
+The term 'API' refers to that code stored in the top level api folder.
+It's important to note that only compliant code & usage will be
+supported. Usage documentation is in the [API
+v3](/api/index.md) page.
+
+## Standards {:#standards}
+
+-   The ONLY supported way to interact with any code in the API folder
+    from outside the api folder is by using the `civicrm_api()` wrapper
+-   Functionality delivered by the API is only supported if it is
+    1.  Advertised via the 'getfields function' OR (preferably AND)
+    2.  Verified by a test
+-   Where functions sit on the api but don't conform to the api
```

Comment:
Hmmm, again, @eileenmcnaughton I'm sorry, but I'm not really sure what you're suggesting with this comment.